internal/auth/jwtauth.go

```diff
@@ -101,18 +101,37 @@ func NewJwtValidator(ctx context.Context, jwksUrl string) (JwtValidator, error)
 	}, nil
 }
 
-var userSubjectContextKey struct{}
+var userTokenContextKey struct{}
 
 // GetUserSubjectFromContext returns the user subject from the context, or nil
 func GetUserSubjectFromContext(ctx context.Context) string {
-	subject, ok := ctx.Value(userSubjectContextKey).(string)
+	token, ok := ctx.Value(userTokenContextKey).(openid.Token)
 	if !ok {
+		fmt.Printf("***\n")
+		fmt.Printf("no token in coGntext\n")
+		fmt.Printf("***\n")
```

Comment:
I'm guessing this is a leftover from debugging?

Reply:
Oops, thanks!